api/src/main/java/org/pmiops/workbench/tools/cdrconfig/CdrConfigVOMapper.java

```diff
@@ -0,0 +1,40 @@
+package org.pmiops.workbench.tools.cdrconfig;
+
+import java.util.List;
+import org.mapstruct.Mapper;
+import org.mapstruct.Mapping;
+import org.pmiops.workbench.db.dao.AccessTierDao;
+import org.pmiops.workbench.db.model.DbAccessTier;
+import org.pmiops.workbench.db.model.DbCdrVersion;
+import org.pmiops.workbench.db.model.DbStorageEnums;
+import org.pmiops.workbench.utils.mappers.CommonMappers;
+import org.pmiops.workbench.utils.mappers.MapStructConfig;
+
+// used by UpdateCdrConfig
+// TODO: enable mapstruct in api/tools and move this class there
+
+@Mapper(
+    config = MapStructConfig.class,
+    uses = {CommonMappers.class, DbStorageEnums.class, AccessTierDao.class})
+public interface CdrConfigVOMapper {
+
+  DbAccessTier toDbTier(AccessTierVO localTier);
+
+  List<DbAccessTier> toDbTiers(List<AccessTierVO> localTiers);
+
+  default List<DbAccessTier> accessTiers(CdrConfigVO cdrConfig) {
+    return toDbTiers(cdrConfig.accessTiers);
+  }
+
+  // currently unambiguous, but adding the qualifiedByName to future-proof
+  @Mapping(source = "accessTier", target = "accessTier", qualifiedByName = "findOneByShortName")
+  @Mapping(source = "archivalStatus", target = "archivalStatusEnum")
+  @Mapping(source = "dataAccessLevel", target = "dataAccessLevelEnum")
```

Comment:
If you forget a field here, or typo one of these - is it a compilation failure? runtime failure? I don't remember.

If this wouldn't currently cause CI to fail, please add a simple test that ensures this happens - I worry about these models going out of sync without some kind of enforcement on presubmit.

Reply:
compile failure.  I think because of `unmappedTargetPolicy = ReportingPolicy.ERROR` in MapStructConfig